IERS test failure if finals2000A.all exists in current working directory

If I create an empty directory, go in it, and download the `finals2000A.all` file:

```
mkdir test
cd test
wget http://maia.usno.navy.mil/ser7/finals2000A.all
python -c 'import astropy; astropy.test(package="utils.iers")'
```

then if I subsequently run the tests, I get the following error:

```
========================================================== FAILURES ===========================================================
___________________________________________________ TestIERS_A.test_simple ____________________________________________________

self = <astropy.utils.iers.tests.test_iers.TestIERS_A object at 0x1164756a0>

    def test_simple(self):
        iers_tab = iers.IERS_A.open()
        jd1 = np.array([2456108.5, 2456108.5, 2456108.5, 2456109.5, 2456109.5])
        jd2 = np.array([0.49999421, 0.99997685, 0.99998843, 0., 0.5])
        ut1_utc, status = iers_tab.ut1_utc(jd1, jd2, return_status=True)
>       assert np.all(status == iers.FROM_IERS_B)
E       assert <function all at 0x1079ac048>(array([-1, -1, -1, -1, -1]) == 0)
E        +  where <function all at 0x1079ac048> = np.all
E        +  and   0 = iers.FROM_IERS_B

../../miniconda3/envs/dev35/lib/python3.5/site-packages/astropy/utils/iers/tests/test_iers.py:139: AssertionError
======================================== 1 failed, 4 passed, 1 skipped in 2.13 seconds ========================================
```

--- a/astropy/utils/iers/tests/test_iers.py
+++ b/astropy/utils/iers/tests/test_iers.py
@@ -16,7 +16,7 @@ from ....time import Time
 FILE_NOT_FOUND_ERROR = getattr(__builtins__, 'FileNotFoundError', OSError)
 
 try:
-    iers.IERS_A.open()  # check if IERS_A is available
+    iers.IERS_A.open('finals2000A.all')  # check if IERS_A is available
 except OSError:
     HAS_IERS_A = False
 else:
@@ -157,6 +157,9 @@ class TestIERS_AExcerpt():
 class TestIERS_A():
 
     def test_simple(self):
+        """Test that open() by default reads a 'finals2000A.all' file."""
+        # Ensure we remove any cached table (gh-5131).
+        iers.IERS_A.close()
         iers_tab = iers.IERS_A.open()
         jd1 = np.array([2456108.5, 2456108.5, 2456108.5, 2456109.5, 2456109.5])
         jd2 = np.array([0.49999421, 0.99997685, 0.99998843, 0., 0.5])

